core/jobs/jobs_manager_test.py

```
@@ -310,3 +307,63 @@ def test_working_voiceover_sync_job(self) -> None:
             beam_job_services.get_beam_job_run_result(run.id).to_dict(),
             {'stdout': 'o', 'stderr': 'e'},
         )
+
+
+class ServiceAccountEmailOptionTests(test_utils.GenericTestBase):
+    @mock.patch('apache_beam.Pipeline', return_value=mock.MagicMock())
+    def test_service_account_email_via_options_raises_error(
+        self, unused_pipeline: mock.Mock
```

Ruff (0.15.17)

[warning] Unused method argument: `unused_pipeline`

(ARG002)